torchrecipes/audio/source_separation/conf/__init__.py

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python3
+import torchrecipes.audio.source_separation.datamodule.librimix  # noqa
+import torchrecipes.core.conf  # noqa
```

Comment:
It's not immediately clear what this initialization is supposed to be doing.
What symbols are supposed to be published here?
It does not look like it's the concern of `audio.source_separation` module to import `core.conf`.

Reply:
Removing the line works fine for training.
Shall we do the same for vision recipes? @tangbinh @kandluis

Reply:
I agree these import statements appear out of no where and we should do something to fix them. Currently, they're there to trigger config registration with Hydra. We're revisiting the config system within `torchrecipes` and hope to address this issue.